packages/opentelemetry-api/package.json

```diff
@@ -11,8 +11,8 @@
     "codecov": "nyc report --reporter=json && codecov -f coverage/*.json -p ../../",
     "codecov:browser": "nyc report --reporter=json && codecov -f coverage/*.json -p ../../",
     "build": "npm run compile",
-    "lint": "gts check",
-    "lint:fix": "gts fix",
+    "lint": "eslint . --ext .ts",
```

Comment:
I am little confuse about this change, AFAIK gts switched from using [tslint](https://palantir.github.io/tslint/) to [eslint](https://eslint.org/) internally. Looks like here we are changing `gts` to `eslint`. Does it mean we dont need gts now?

Reply:
we use it ref: https://github.com/google/gts#working-with-eslint 
https://github.com/open-telemetry/opentelemetry-js/pull/892/files#diff-6759c3472b4d1706bb3ffa02871d49f1R12

Reply:
I actually prefer this way because my eslint plugin for vs code works much better